src/components/StatisticBlock.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { FC, ReactNode } from 'react'
+import styled from 'styled-components'
+
+import LoadingSpinner from './LoadingSpinner'
+
+const Block = styled.div`
+  background-color: ${({ theme }) => theme.bgTertiary};
+  border: 1px solid ${({ theme }) => theme.borderSecondary};
+  box-shadow: ${({ theme }) => theme.shadowPrimary};
+  border-radius: 7px;
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  padding: 2rem;
+`
+
+const Title = styled.div`
```

Comment:
So, some "conventions" we've agreed on with @nop33 (conventions, yay! :D) :
- We always put the styled components after the main exported react component (which is `StatisticBlock` here)
- We put the types / interface first.

So we always have 1) Prop types 2) exported component 3) styling.

Reply:
o7 roger that